addons/hr_timesheet/views/hr_timesheet_views.xml

```diff
@@ -88,6 +112,28 @@
             </field>
         </record>
 
+        <record id="view_hr_timesheet_line_graph_groupby_project" model="ir.ui.view">
+            <field name="name">account.analytic.line.graph</field>
+            <field name="model">account.analytic.line</field>
+            <field name="inherit_id" ref="view_hr_timesheet_line_graph"/>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <field name="task_id" position="replace"/>
+            </field>
+        </record>
```

Comment:
@xavierbol , If we remove the `task_id` from the parent view then it will also impact the graph view of the timesheet from the invoice which is grouped by project and task(invoice >> timesheet button action: `action_timesheet_from_invoice`). that's why we have not updated the parent view.

Thanks